AfricanReligions/events/afr_misc_events.txt

```diff
@@ -489,4 +489,230 @@ character_event = {
 
         hidden_tooltip = { character_event = { id = AFRREL.052 } }
     }
+}
+
+# Amazigh Selecting Patron Deities events
+character_event = {
+    id = AFRREL.060
+    desc = EVTDESC_AFRREL060
+    border = GFX_event_normal_frame_religion
+    picture = GFX_evt_afr_amazigh_deities_main
+
+    is_triggered_only = yes
+
+    immediate = {
+        if = {
+            limit = { is_playable = yes }
+            any_courtier = {
+                limit = {
+                    prisoner = no
+                    OR = {
+                        religion = amazigh
+                        religion = amazigh_reformed
+                    }
+                    age = 16
+                    has_minor_deity_trigger = no
+                    NOT = { is_playable  = yes }
+                }
+                character_event = { id = AFRREL.060 days = 1 }
+            }
+        }
+    }
+
+    option = {
+        name = EVTOPTA_AFRREL060
+        hidden_tooltip = { character_event = { id = AFRREL.061 } }
+    }
+    option = {
+        name = EVTOPTB_AFRREL060
+        hidden_tooltip = {
+            piety = 25
+            remove_character_modifier = picked_main_deity_timer
+        }
+    }
+}
+
+character_event = {
+    id = AFRREL.061
+    desc = EVTDESC_AFRREL061
+    border = GFX_event_normal_frame_religion
+    picture = GFX_evt_afr_amazigh_deities_main
+
+    is_triggered_only = yes
+
+    option = {
+        name = EVTOPTA_AFRREL061
+
+        hidden_tooltip = { character_event = { id = AFRREL.063 } }
+    }
+
+    option = {
+        name = EVTOPTB_AFRREL061
+
+        hidden_tooltip = { character_event = { id = AFRREL.064 } }
+    }
+
+    option = {
+        name = EVTOPTC_AFRREL061
+
+        hidden_tooltip = { character_event = { id = AFRREL.065 } }
+    }
+
+    option = {
+        name = EVTOPTD_AFRREL061
+
+        hidden_tooltip = { character_event = { id = AFRREL.062 } }
+    }
+}
+
+character_event = {
+    id = AFRREL.062
+    desc = EVTDESC_AFRREL061
+    border = GFX_event_normal_frame_religion
+    picture = GFX_evt_afr_amazigh_deities_main
+
+    is_triggered_only = yes
+    
+    option = {
+        name = EVTOPTA_AFRREL062
+
+        hidden_tooltip = { character_event = { id = AFRREL.066 } }
+    }
+
+    option = {
+        name = EVTOPTB_AFRREL062
+
+        hidden_tooltip = { character_event = { id = AFRREL.067 } }
+    }
+
+    option = {
+        name = EVTOPTC_AFRREL062
+
+        hidden_tooltip = { character_event = { id = AFRREL.060 } }
```

Comment:
This goes to the wrong event.  It should go to 61.

Reply:
I had it go back to the opening event of the chain so you could then not choose any patron deity if you wanted to and exit the event chain but if this is incorrect I'll change it

Reply:
I think that'd be a rare case.  The vanilla events don't even have a choice to exit the event chain, I don't think.